helpers/index_db.php

```diff
@@ -2,12 +2,16 @@
 
 require_once __DIR__ . "/lib/index_db.inc.php";
 
-function myLog(string $level, string $msg): void {
-    echo "$level : $msg \n";
-}
+$Loggers->addLogger(new class implements LoggerSubscriber{
+    public function log(string $type, string $message){
+        echo "$type : $message \n";
+    }
+});
 
 function run(): void {
-    myLog('info', 're-creating index');
+    global $Loggers;
```

Comment:
Better stay away from global. This might cause a bunch of problems in the long run.

Reply:
so how should i refernce the logger instead?